### DISCUSSION TITLE
Login forbidden?

### DISCUSSION THREAD

**Discussion by RonRN18** [tgorgdotcom/locast2plex#282] [General]
I started using this app many months ago, running this in a VM. I then put my project on hold. I have since returned to playing with it and I'm not being able to start properly. I have updated my VM (running Ubuntu Server 21.04) with the latest git pull. Below is my config.ini file with a few redactions and obfuscations, but I'm showing it as a template for how it appears.

`
[main]
locast_username = my@email.com
locast_password = MyPassword
plex_accessible_ip = xxx.xxx.xxx.xxx
plex_accessible_port = 6077
override_latitude = 33.6683374
override_longitude = -117.9933135
fcc_delay = 43200

`

When I try to start locast2plex, I get the following:

`Locast2Plex v0.5.3
Tuner count set to 3
UUID set to: XXXXXX...
Logging into Locast using username my@email.com...
Error in function login: Forbidden
Exiting...
'

I have logged in via the Locast website just to ensure there wasn't anything wrong with the log in process. I'm not sure why my login in "Forbidden". Could it have something to do with the addition of "Profiles"? That is the only thing I noticed different when logging in via the website.

**Comment by RonRN18** [comment]
I had performed a 'git pull' to update but apparently it didn't update properly. When I realized this. I ended up coping my 'config.ini' to a temporary directory elsewhere, I deleted my locast2plex directory and re-cloned from scratch. I then copied in my config.ini file I had copied elsewhere into the locast2plex directory and that solved the issue.

**Reply by cahiatt** [reply]
I just had the same issue but for me nothing had changed. On server reboot for a different update it was giving the login forbidden error.  I did a docker-compose pull request for an update and everything is working again.
